package.json

```diff
@@ -26,7 +26,7 @@
     "eslint": "^7.32.0",
     "eslint-config-airbnb-base": "^14.2.1",
     "eslint-plugin-import": "^2.26.0",
-    "hint": "^6.2.0",
+    "hint": "^2.0.0",
```

Comment:
Hello @shahira-sadat 
Great job so far implementing the project requirements.

- [ ] Your app throws an error when I try to start it with webpack `npm run start`. Indicating that you are missing some `package.json` dependencies.
![solutio](https://user-images.githubusercontent.com/55596970/179629813-1e159ee3-63d7-492c-9e16-5c4698132ffd.png)
### **SOLUTION**
You can install the missing dependencies by running this command on your terminal from your project root directory.
`npm install --save-dev babel-loader @babel/core @babel/preset-env webpack`